backend/src/main/java/ch/puzzle/okr/ForwardFilter.java

```diff
@@ -1,25 +1,24 @@
 package ch.puzzle.okr;
 
+import java.io.IOException;
 import jakarta.servlet.FilterChain;
 import jakarta.servlet.ServletException;
 import jakarta.servlet.ServletRequest;
 import jakarta.servlet.ServletResponse;
 import jakarta.servlet.http.HttpServletRequest;
+
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
-import org.springframework.stereotype.Component;
 import org.springframework.web.filter.GenericFilterBean;
 
-import java.io.IOException;
-import java.util.Arrays;
-
 public class ForwardFilter extends GenericFilterBean {
 
     private static final Logger logger = LoggerFactory.getLogger(ForwardFilter.class);
 
     @Override
     public void doFilter(ServletRequest servletRequest, ServletResponse servletResponse, FilterChain filterChain)
-            throws IOException, ServletException {
+            throws IOException,
+            ServletException {
```

Comment:
this seems a bit weird, why is servlet exception on a new line, and the bodi is less indented than the declaration?